contracts/apps/AragonApp.sol

```diff
@@ -30,4 +31,10 @@ contract AragonApp is AppStorage, Initializable, ACLSyntaxSugar, EVMScriptRunner
         }
         return address(kernel) == 0 || kernel.hasPermission(_sender, address(this), _role, how);
     }
+
+    function getRecoveryVault() public view returns (address) {
+        // Funds recovery via a vault is only available when used with a kernel
+        require(address(kernel) != 0);
```

Comment:
I've just disabled this functionality for apps that don't have a kernel, as the easiest way out.

If users use `AragonApp` by itself and want an escape hatch, they can implement their own.